test/errors.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/**
+ * Tests expecting promises to reject
+ */
+
+var cheerio = require('cheerio');
+var meta = require('../index');
+var preq = require('preq'); // Promisified Request library
+var assert = require('./utils/assert.js');
+
+
+// mocha defines to avoid JSHint breakage
+/* global describe, it, before, beforeEach, after, afterEach */
+
+
+describe('errors', function() {
+
+	var url = 'http://www.example.com/';
+//	TODO: ALL BROKEN- Pass even when they shouldn't i.e. uncomment following line:
+//	var url = 'http://blog.woorank.com/2013/04/dublin-core-metadata-for-seo-and-usability/';
```

Comment:
Still getting unhandled rejections that aren't failing the tests from assert.fails here.